### ISSUE TITLE
[Ballista] Ballista common issues / bugs list

### ISSUE THREAD

**Issue by ThoseWhoFight** [EdenServer/community#5217]
- [ ] I have searched for existing issues for issues like this one. The issue has not been posted.
- [ ] I have provided reproducible steps. (No "as the title says" posts please. ***Please try to state facts only, and avoid assumptions/opinions.***)
- [ ] I have provided my client version in the details. (type /ver into your game window)
- [ ] ***I have provided proof of my claim (era videos, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

### Details
Consolidated list of common issues and bugs plaguing ballista matches currently, by the participants comprised of Eden/Retail players.

* Spell duration is too long and doesn't diminish when constantly getting hit with the same spell multiple times.
(Don't think it should be longer than 30 seconds(?) unlike PvE. The "potency" should reset back to normal after HPing or Raising. Does only apply to CC spells like bind, gravity, sleep, ect.)

* Range attacks still hits even when you run out of range and should whiff(?)
[Relevant retail video](https://www.youtube.com/watch?v=OTamkmVv3uI)

* Gravity potency is too strong, it's PvE % gravity. Should be set to slow running speed.

* Coming back with Full TP instead of 500 when you raise back up.

* MP should be restored to half MP after letting reraise timer finish, if under half MP. No MP should be restored if 50% or over(?)

* Poison Pot are hard to find when they should be a higher % on finding one

* Suzaku Sune-ate blaze spikes damage shows but doesn't actually do damage

* (Needs more confirmation) Enspell only procs on the first hit of a multi-attack, and the rest shows the damage but doesnt even do the damage

* Using items with a long casting time (ex. Hi elixir) will finish the casting bar and disappear but the item is still casting.

* Post death invisibility should not take any damage from AoE. Player's so be invulnerable during this phase 

* Lethe Water +1 appears to not be a usable item, NQ version seemed fine.

* If matches are uneven in players there should be more than one rook.

* Outside items such as food are still usable via /item "item name" macros in scenarios where they should not be, such as Pashhow Marshlands and parts of Meriphataud Mountains matches.

**Comment by ThoseWhoFight** [comment]
Adding two YT channels for retail ballista video resources

https://www.youtube.com/user/NatealityTV

https://www.youtube.com/channel/UCPFXMQGpl7XQimEV27RQj_A

**Comment by 59blargedy** [comment]
thanks for pulling these together. some qs/comments

- the spell thing already exists and occurs currently on spell sleeps, i see where things are broken for other pieces. 
- lethe water hq working fine for me, can you double check?
- would appreciate some proof on gravity, tp/mp, just dont have the time to scour vids
- ranged attacks are set to whiff already past range. need double check/ss
- unclear on the item usage piece, is the issue that the casting bar differs from the actual use time?
- poison pots are deliberately called out with a high rate as is, wondering if this is just bad luck on someone's part so they got an impression they dont drop enough. could raise it, but would want a baseline to shoot for.

**Comment by 59blargedy** [comment]
another general q i have that needs retail research:
if a bard uses troubadour to extend the length of an enfeebling spell, does the duration cap occur before or after (i.e will troubadour allow double the max time, or just be capped and therefore be basically useless)

**Comment by ThoseWhoFight** [comment]
So I think I figured this one out possibly, it happened again during the Meriphataud match earlier using a demoralizer this time. I noticed the casting got interrupted using it on an enemy team member, I still had the item in inventory but could no longer use it and I could no longer drop it either. Guessing maybe it registered as if the item went off, but it didn't and could no longer be tossed either.

**Comment by ThoseWhoFight** [comment]
Additionally, found that certain actions like Provoke and missed ranged attacks didn't seem to interrupt an enemy team member while mid scoring.

**Comment by 59blargedy** [comment]
> I noticed the casting got interrupted using it on an enemy team member, I still had the item in inventory but could no longer use it and I could no longer drop it either.
was the casting interrupted bc they ran out of range or the item would be interrupted regardless upon use?

**Comment by ThoseWhoFight** [comment]
Reviewing the footage, it's because he stepped out of range of the item being used the first time. After that it just gave the message "Unable to use item" and "Drop item" was greyed out.

**Comment by ThoseWhoFight** [comment]
- Suzaku Sune-ate blaze spikes damage shows but doesn't actually do damage

Also noticed this wasn't exclusive to pvp, seems to proc in chat log vs pve mobs as well, but 0 damage is actually done. Not sure if that helps at all.

**Comment by inaringo** [comment]
Couple more issues in ballista:
-Third Eye should block ranged attacks and ranged WS
-Carnal Incense should block ranged WS as long as they are physical

**Comment by inaringo** [comment]
https://www.nicovideo.jp/watch/sm37018270 @2:05 Kilin on the Windurst team gets hit with Gravity. You can see the slow run @2:14.

**Comment by inaringo** [comment]
https://www.youtube.com/watch?v=VyNLYdWaGj0

6:11 Niiome anticipates a ranged attack with Third Eye. Not sure if too recent to count, but it seems Third Eye was never changed after era cut-off.

**Comment by KOS-elos** [comment]
@30:17 TP was 38%. Got hit a lot and died at 30:59. Didn't range anyone and shows he has exactly 50% TP 32:01. Can't tell if its just crazy coincidence he got exactly 50% when he died, or the statements are true that when you auto revive you do indeed get 50% back only. https://www.nicovideo.jp/watch/sm37018419

**Comment by inaringo** [comment]
Kabura arrows don't seem to be subject to wind resist checks when they should be and always land for full duration of 60 seconds when it should be shorter. Was getting hit 100% and full duration through +40 wind resist equipment, barsilence, and baraero.

**Comment by 59blargedy** [comment]
> Silence from Kabura Arrows don't seem to be subject to wind resist checks when they should be and always land for full duration of 60 seconds when it should be shorter. Was getting hit 100% and full duration through +40 wind resist equipment, barsilence, and baraero.

weapon addl effects are pass fail. either it will land for full duration or it wont land, and it goes through resistance checks that include barspells, wind resist, etc to determine this. if the duration of addl weapon effects should be shortened as a general thing in ballista (like casted enfeebles) then that would be an issue, but your wind resist will not affect duration.

**Comment by ThoseWhoFight** [comment]
Adding status ammo audit to list. Sleep bolts definitely refresh endlessly with little to no variance, unsure if this is accurate. Need retail evidence

**Comment by 59blargedy** [comment]
> Adding status ammo audit to list. Sleep bolts definitely refresh endlessly with little to no variance, unsure if this is accurate. Need retail evidence
sorry, just trying to understand what refresh endlessly means, can you clarify? do you mean the sleep bolts overwrite? also, kinda same q as with inaringo, unsure if their duration should be affected in any way.

**Comment by ThoseWhoFight** [comment]
> i

Yes, sleep bolts overwrite to full duration. Unsure how accurate this is, but doesn't seem like it's right. Just making a note so people can find evidence when looking through retail videos or if a retail player can test it would be pretty open shut case I imagine.

**Comment by KOS-elos** [comment]
https://youtu.be/JgVncTOeHHk
Can someone clarify if this may be due to the lag difference, that Octo still got hit when he was out of range?

**Comment by 59blargedy** [comment]
resolved tp, gravity, ra range in October 7, 2022 update. would be helpful if op was updated to note resolved items. thank you.

**Comment by ThoseWhoFight** [comment]
Eden testing:
**Provoke / Pebble missing / All ranged attacks hit or miss doesn't interrupt scoring**
aoe damage (battle dance) interrupts properly
aoe non damage (poisonga) interrupts properly
Cure spells from allies interrupts properly
single target non dmg (Para/Slow) interrupts properly

Seems to just be JAs and ranged attacks that needs fixing currently.

**Comment by ThoseWhoFight** [comment]
Additionally,
no variable resist or gradual resist on sleep bolts, static 24 seconds or full resist(no sleep proc) but nothing in between. Timer should decrease after multiple procs in short period. Retail testing needed?

**Comment by SolidSioux** [comment]
Idk if this helps or will be considered enough proof but this video shows a Samurai anticipating a ranged attack in Ballista and in era. Happens at the one minute mark.

https://www.youtube.com/watch?v=Sqa2Jewoh14&ab_channel=Thienthan

**Comment by inaringo** [comment]
Leviathan Tail whip gives full potency (instead of ballista potency) gravity plus ~2min duration when it should be less.

**Comment by inaringo** [comment]
Physical AOE blue magic, such as Battle Dance and Blastbomb, should remove Third Eye.

**Comment by 59blargedy** [comment]
do you have a source? otherwise ill need to test it myself

**Comment by inaringo** [comment]
nah I'll see if I can find something tho. just something multiple people have told me.

**Comment by inaringo** [comment]
https://wiki.ffo.jp/html/3389.html

So it doesn't specify BLU magic, but English and Japanese wiki do say that AOE physical moves do damage through and remove Third Eye. Also Japanese wiki mentions Third Eye began absorbing ranged attacks in 2005, and specifically mentions Ballista as an example.

**Comment by 59blargedy** [comment]
ok, i'll check blue magic specifically, thank you. we're good on proof for ranged attack being absorbed by third eye, it's just that it is difficult to code.

**Comment by inaringo** [comment]
> Leviathan Tail whip gives full potency (instead of ballista potency) gravity plus ~2min duration when it should be less.

https://youtu.be/mwl8PLdgeNg?t=2863 As video shows-- Gravity from Tail Whip puts you at walk speed when it should be slow run.
